azure-pipelines.release.yml

```diff
@@ -0,0 +1,63 @@
+pr: none
+trigger: none
+
+variables:
+  - group: "Github and NPM secrets"
+
+pool:
+  vmImage: "Ubuntu 16.04"
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: "8.x"
+    displayName: "Install Node.js"
+
+  - script: |
+      git config user.name "UI Fabric Build"
+      git config user.email "fabrictactical@service.microsoft.com"
+      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/OfficeDev/office-ui-fabric-react.git
+    displayName: Authenticate git for pushes
+
+  - script: |
+      npm install --ignore-scripts
+      node node_modules/@microsoft/rush/bin/rush install --bypass-policy
+    displayName: rush install
+
+  - script: |
+      npm run build
+    displayName: rush build (Create production build)
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: apps/fabric-website/dist
+      artifactName: "fabric-website"
+      publishLocation: "Container"
+    displayName: "Publish Artifact: Fabric Website"
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: packages/office-ui-fabric-react/dist
+      artifactName: "fabric"
+      publishLocation: "Container"
+    displayName: "Publish Artifact: Fabric"
+
+  - script: |
+      node node_modules/@microsoft/rush/bin/rush publish -a -p -b 5.0 -n $(npmToken) --add-commit-details --set-access-level public
```

Comment:
I think this needs a tag or else would be published to latest...

Reply:
Is `legacy-5.0` okay?